src/com/company/coffe/Espresso.java

```diff
@@ -0,0 +1,13 @@
+package com.company.coffe;
+
+
+import java.util.HashMap;
+
+public class Espresso extends CoffeAbstract implements Coffe {
+    private Integer water = 200;
```

Comment:
Why you override abstract class fields? It's not a good way to use an abstraction, please try to resolve this exercise without OOP, because you not pass this topic yet